Document the need to use quotes around Kubernetes version (#9175)
aws/eks-anywhere: docs/content/en/docs/whatsnew/changelog.md

---

{{% alert title="Announcements" color="warning" %}}

> * Due to a bug in the `sigs.k8s.io/yaml` module that EKS Anywhere uses, Kubernetes versions whose minor versions are multiples of 10, such as `1.30`, `1.40`, etc, will be parsed as `float64` instead of `string` if specified without quotes in the cluster config file, which causes the trailing zero to get dropped, thereby incorrectly passing `1.3` and `1.4` to downstream operations. The short-term solution is for users to use single or double quotes around their Kubernetes version(s) whose minor versions are multiples of 10.

Reviewer: Is this better?
```suggestion
* Due to a bug in the `sigs.k8s.io/yaml` module that EKS Anywhere uses, Kubernetes versions whose minor versions are multiples of 10, such as `1.30`, `1.40`, etc, will be parsed as `float64` instead of `string` if specified without quotes in the cluster config file. This causes the trailing zero to get dropped and be evaluated as `1.3` and `1.4` respectively. For now, use single or double quotes around their Kubernetes version(s) whose minor versions are multiples of 10.
```

Author: Yeah! change, thanks 🚀
